src/custom/dep.cpp

```diff
@@ -291,6 +296,12 @@ static inline bool invertPaint(NSVGshape* const shape, NSVGpaint& paint, const c
     if (paint.type != NSVG_PAINT_COLOR)
         return false;
 
+    // Special case for AaronStatic orange color
```

Comment:
a whole check for it is not wanted. just put this color a bit further down where all the ignored colors are.

Reply:
hmm. I figured if this is used in any other module maybe it shouldn't be touched. moved it to the other list then.

Reply:
true, but typically colors outside of black/grey/white shouldnt be inverted.
we could in theory define a range of colors and it would work for this too, but for now hardcoding also works.